event-log-service-webapp/src/main/resources/sql-scripts/copyOfflineConcepts.py

```diff
@@ -1,15 +1,33 @@
 import MySQLdb
 import uuid
 import datetime
+import os
+import getpass
 
-db = MySQLdb.connect("localhost","root","password","openmrs")
+
+mySqlPswd = os.environ.get('MYSQL_ROOT_PASSWORD');
+
+if 'None' == str(mySqlPswd) or '' == str(mySqlPswd):
+    try:
+       mySqlUser = raw_input('MySQL user: ')
+       mySqlPswd = getpass.getpass(prompt='MySQL password: ')
+    except Exception as error:
+       print 'ERROR: '+ error;
+    else:
+       print 'MySQL user: '+ mySqlUser
+       print 'MySQL password: '+ mySqlPswd;
+else:
+    mySqlUser = 'root'
```

Comment:
Printing of username and password is not required, message can just say that connected with xyz user.
1) Original script can connect only to root user with given password.
2) current script has 2 scenarios, when environment variable not mentioned it can connect to any user with given password, if env is specified it will always connect with root user.
so we can prompt only for password for root user.